cmd/glasskube/cmd/version.go

```diff
@@ -0,0 +1,77 @@
+package cmd
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/glasskube/glasskube/internal/config"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+var versioncmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print the version of glasskube and package-operator",
+	Long:  `Print the version of glasskube and package-operator`,
+	Run: func(cmd *cobra.Command, args []string) {
+		glasskubeVersion := config.Version
+		fmt.Fprintf(os.Stderr, "glasskube: v%s\n", glasskubeVersion)
+		operatorVersion, err := getPackageOperatorVersion()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "✗ no deployments found in the glasskube-system namespace\n")
+		} else {
+			fmt.Fprintf(os.Stderr, "package-operator: %s\n", operatorVersion)
+		}
+	},
+}
+
+func init() {
+	RootCmd.AddCommand(versioncmd)
+}
+
+func getPackageOperatorVersion() (string, error) {
+	var kubeconfig string
+	if home := homedir.HomeDir(); home != "" {
+		flag.StringVar(&kubeconfig, "kubeconfig", home+"/.kube/config", "(optional) absolute path to the kubeconfig file")
+	} else {
+		flag.StringVar(&kubeconfig, "kubeconfig", "", "absolute path to the kubeconfig file")
+	}
+	flag.Parse()
+
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	if err != nil {
+		return "", err
+	}
```

Comment:
We already have a `PreRun` util for this, which can be used by adding `PreRun: cliutils.SetupClientContext(true)` to the cmd.

The kubeconfig flag already exists on the root cmd, so you don't have to implement your own.